agent/php_explain_pdo_mysql.c

```diff
@@ -130,7 +130,11 @@ nr_explain_plan_t* nr_php_explain_pdo_mysql_statement(zval* stmt,
     goto end;
   }
 
+#if ZEND_MODULE_API_NO >= ZEND_8_5_X_API_NO
+  dbh = pdo_stmt->database_object_handle;
+#else
   dbh = &pdo_stmt->database_object_handle;
+#endif
   dup = nr_php_pdo_duplicate(dbh TSRMLS_CC);
```

Comment:
It needs a null check somewhere along the path for `dbh`. 
`dbh` is passed into `nr_php_pdo_duplicate`, which for [8.5 passes](https://github.com/newrelic/newrelic-php-agent/blob/828fb8ee72a1a5fc2735d9be003b347f7d8a1a04/agent/php_pdo.c#L420C13-L420C60) to `nr_php_pdo_get_database_object_from_zend_object` which dereferences [here](https://github.com/newrelic/newrelic-php-agent/blob/828fb8ee72a1a5fc2735d9be003b347f7d8a1a04/agent/php_pdo.c#L191).

Reply:
A null check has been added to `nr_php_pdo_get_database_object_from_zend_object` before dereferencing here: [5581a2c](https://github.com/newrelic/newrelic-php-agent/commit/5581a2c804e77871c09bd8246891aef25d809f6d)